gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -220,6 +220,61 @@ static String selector(LicenseField... fields) {
   }
 
   /**
+   * Fields of a Compute Engine Operation resource.
+   *
+   * @see <a
+   *     href="https://cloud.google.com/compute/docs/reference/latest/globalOperations#resource">
+   *     GlobalOperation Resource</a>
+   * @see <a
+   *     href="https://cloud.google.com/compute/docs/reference/latest/regionOperations#resource">
+   *     RegionOperation Resource</a>
+   * @see <a href="https://cloud.google.com/compute/docs/reference/latest/zoneOperations#resource">
+   *     ZoneOperation Resource</a>
+   */
+  enum OperationField {
+    CLIENT_OPERATION_ID("clientOperationId"),
+    CREATION_TIMESTAMP("creationTimestamp"),
+    DESCRIPTION("description"),
+    END_TIME("endTime"),
+    ERROR("error"),
+    HTTP_ERROR_MESSAGE("httpErrorMessage"),
+    HTTP_ERROR_STATUS_CODE("httpErrorStatusCode"),
+    ID("id"),
+    INSERT_TIME("insertTime"),
+    NAME("name"),
+    OPERATION_TYPE("operationType"),
+    PROGRESS("progress"),
+    SELF_LINK("selfLink"),
+    START_TIME("startTime"),
+    STATUS("status"),
+    STATUS_MESSAGE("statusMessage"),
+    REGION("region"),
```

Comment:
Nit: "region" goes before "self_link" (I only point it out because you it looks like you were alphabetizing this ginormous list of fields)